plugins/user-authenticators/ldap/src/org/apache/cloudstack/api/command/LdapImportUsersCmd.java

```diff
@@ -70,10 +72,12 @@
 
     @Parameter(name = ApiConstants.ACCOUNT_TYPE,
                type = CommandType.SHORT,
-               required = true,
```

Comment:
@jburwell see here ^^, accountype was a required arg but with our changes we've made it non-required also added the roleId arg. Therefore, at least one of the two must be provided, if both are provided we consider roleId (and account type of the role).